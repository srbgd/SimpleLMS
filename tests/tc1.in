

drop
register librarian --login l --password pass --name test --address innopolis --phone-number 88005553535 --card-number 1234
register student --login p --password pass --name test --address innopolis --phone-number 88005553535 --card-number 9876
insert book --title b title --authors unknown --publisher empty --edition none --date unknown --price 1000
copy 2
copy 2
login
p
pass
find
checkout 2
login
l
pass
check
exit
