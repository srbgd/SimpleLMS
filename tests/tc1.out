Success
True
True
True
True
True
True
Success
[{'attributes': {'address': 'innopolis', 'phone-number': '88005553535', 'password': 'pass', 'card-number': '1234', 'name': 'test', 'login': 'l'}, 'type': 'librarian', 'id': 0}, {'attributes': {'address': 'innopolis', 'phone-number': '88005553535', 'password': 'pass', 'card-number': '9876', 'name': 'test', 'login': 'p'}, 'type': 'student', 'id': 1}, {'attributes': {'edition': 'none', 'authors': 'unknown', 'publisher': 'empty', 'title': 'b title', 'date': 'unknown', 'price': '1000'}, 'type': 'book', 'id': 2}, {'attributes': {'deadline': '', 'user_id': None, 'origin_id': '2'}, 'type': 'copy', 'id': 3}, {'attributes': {'deadline': '', 'user_id': None, 'origin_id': '2'}, 'type': 'copy', 'id': 4}]
True
Success
[{'attributes': {'deadline': '27/02/2018', 'user_id': 1, 'origin_id': '2'}, 'type': 'copy', 'id': 3}, {'attributes': {'deadline': '', 'user_id': None, 'origin_id': '2'}, 'type': 'copy', 'id': 4}]
